Api/IriConverter.php

```diff
@@ -134,4 +143,22 @@ private function getRouteName(ResourceInterface $resource, $prefix)
             }
         }
     }
+
+    /**
+     * @param ResourceInterface $resource
+     * @param array $context
+     *
+     * @return string
+     */
+    private function getIdentifierFromResource(ResourceInterface $resource, array $context = array())
```

Comment:
Yes but the argument is never passed in parameter in that class and the method is private. So this code is unreachable.